Multisig/Data/Services/Utils/GSError.swift

```diff
@@ -351,6 +351,51 @@ enum GSError {
     }
 
 
+    struct InvalidWalletConnectQRCode: DetailedLocalizedError {
+        let description = "Can’t use this QR code"
+        let reason = "Scaned QR code is not in a format that we expect."
+        let howToFix = "Please assure that the displayed QR code is in a valid format"
+        let domain = clientErrorDomain
+        let code = 1111
```

Comment:
iOS-specific codes should start with 13

Reply:
For WalletConnect prototype error codes will start with 99xx

Reply:
added everything to the doc